modules/end-user-guide/partials/proc_creating-workspaces.adoc

```diff
@@ -2,4 +2,223 @@
 [id="creating-workspaces"]
 = Creating workspaces
 
-Rough technical info to be dumped here by the assigned SME.
+If your use case does not permit use of the {prod-short} dashboard, you can create workspaces with {orch-name} APIs by applying custom resources to the cluster.
+
+[NOTE]
+====
+
+Using the {prod-short} dashboard is the default way to create workspaces because {prod-short} and its components automatically convert the target Git repository's devfile into the `DevWorkspace` and `DevWorkspaceTemplate` custom resources on the cluster with the following configuration benefits:
+
+* Recognition of configurations in `spec.devEnvironments` specified in the `CheCluster` custom resource including:
+** Persistent storage strategy specified with `devEnvironments.storage`
+** Default IDE specified with `devEnvironments.defaultEditor`
+** Default plugins specified with `devEnvironments.defaultPlugins`
+** Container build configuration specified with `devEnvironments.containerBuildConfiguration`
+* Recognition of the `DevWorkspaceOperatorConfig` configuration managed by {prod-short}
+
+====
+
+.Prerequisites
+
+* An active `{orch-cli}` session with permissions to create `DevWorkspace` and `DevWorkspaceTemplate` resources in your {orch-namespace} on the cluster. See {orch-cli-link}.
+
+* You know the relevant {prod-short} user namespace on the cluster.
++
+TIP: You can visit `pass:c,a,q[{prod-url}]/api/kubernetes/namespace` to get your {prod-short} user namespace as `name`.
+
+* You are on the {prod-short} user namespace on the cluster.
++
+[TIP]
```

Comment:
This tip is not useful in this context (i.e. in all the commands below the namespace is explicitly specified). And, if the user wants to change the current OpenShift project, he should be able to learn it from the OpenShift documentation, I would not expect Eclipse Che documentation to explain that. @max-cx can we remove that?

Reply:
@l0rd, sure. I've had a look at it and I'll remove it from this procedure.